<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ByteOpsAction</class>
 <widget class="QWidget" name="ByteOpsAction">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>31</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1,0,0,0">
   <property name="margin">
    <number>4</number>
   </property>
   <item>
    <widget class="QLabel" name="lblText">
     <property name="text">
      <string>Byte Operations:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="SpinBoxBaseSelector" name="sbbsValue" native="true"/>
   </item>
   <item>
    <widget class="QComboBox" name="cbByteOps">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pbExecute">
     <property name="text">
      <string>Execute</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pbClose">
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SpinBoxBaseSelector</class>
   <extends>QWidget</extends>
   <header>spinboxbaseselector/spinboxbaseselector.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
